Frontend/src/pages/registrations/components/RegistrationList.jsx

```diff
@@ -10,40 +10,30 @@ import styles from './list.module.scss'
 
 function sortReducer(state, action) {
   if (action.type === 'CHANGE_SORT') {
-    if (state.column === action.column) {
+    if (state.sortColumn === action.sortColumn) {
       return {
         ...state,
-        data: state.data.slice().reverse(),
-        direction: state.direction === 'ascending' ? 'descending' : 'ascending',
+        sortDirection:
+          state.sortDirection === 'ascending' ? 'descending' : 'ascending',
       }
     }
-    switch (action.column) {
+    switch (action.sortColumn) {
```

Comment:
All these cases are identical, so all the existing cases can be a single branch of the switch. Or if we're ok removing the "unknown column" error then we could just remove the switch entirely and
```
      return {
        sortColumn: action.sortColumn,
        sortDirection: 'ascending',
      }
```

Reply:
I kept the switch and just made the cases fall through.